Easy.Common/RestClient.cs

```diff
@@ -351,6 +350,18 @@ public void Dispose()
             _client.Dispose();
             lock (_endpoints) { _endpoints.Clear(); }
         }
+        
+        /// <summary>
+        /// Makes the necessary changes to <see cref="ServicePointManager" /> properties.
+        /// </summay>
+        public static void ConfigureServicePointManager()
```

Comment:
I've made it public just in case user would want to re-apply these changes by some reason

Reply:
I prefer it to remain `private`. If anyone wants to override they can do so outside the context of the `RestClient` or wrap it inside their own abstraction.

Reply:
Fair enough, reverted